screens/authentication/signUp/SignUpScreen.jsx

```diff
@@ -99,61 +99,47 @@ const SignUpScreen = ({navigation}) => {
           <Formik>
             <View style={styles.inputsContainer}>
               <View style={styles.nameInputStyle}>
-                <TextInput
+                <CustomTextInput
                   name="name"
                   placeholder="Name"
-                  placeholderTextColor={theme.colors.cool_gray}
-                  style={styles.input}
-                  variant="standard"
                   onChangeText={handleChange('name')}
                   onBlur={() => setFieldTouched('name')}
```

Comment:
instead of setFieldTouched there is an onBlur function you can get from useFormik